report/renderer/details-renderer.js

```diff
@@ -394,29 +496,48 @@ export class DetailsRenderer {
       this._dom.createChildOf(theadTrElem, 'th', classes).append(labelEl);
     }
 
+    const entityItems = this._getEntityGroupItems(details);
```

Comment:
It'd be helpful to come up with a name for the table item we use for the entity descriptor row, and use it throughout `_renderEntityGroupRow` / `_adornTableRowWithEntityChips` / this function

Reply:
One thing to note is that all render* prefixed functions return a DOM/DocumentFragment. I think it might be better off calling this `_getEntityGroup...Items`